core/wiki/macros/tag-picker.tid

```diff
@@ -34,13 +39,15 @@ $actions$
 <div class="tc-block-dropdown-wrapper">
 <$reveal state=<<qualify "$:/state/popup/tags-auto-complete">> type="nomatch" text="" default="">
 <div class="tc-block-dropdown">
+<$list filter="[{$:/temp/NewTagName}minlength{$:/config/Tags/MinLength}limit[1]]" emptyMessage="""<div class="tc-search-results">{{$:/language/Search/Search/TooShort}}</div>""" variable="listItem">
 <$list filter="[tags[]!is[system]search:title{$:/temp/NewTagName}sort[]]" variable="tag">
```

Comment:
Tto me, there should be (at least) a `limit[50]` after the search... everything else is impractical and slow (for me).